tests/unit/version/molgenis_version_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import pytest
+
+from mcmd.utils.errors import McmdError
+from mcmd.version import molgenis_version
+
+
+@pytest.mark.unit
+class MolgenisVersionTest(unittest.TestCase):
```

Comment:
what is the expected behavior for versions in other molgenis app repositories such as 7.0.0-BBMRI-ERIC?

Reply:
See the second to last test case: 

```python
def test_extract_version_number_multi():
        version = molgenis_version._extract_version_number('7.2.3-APP-1.0.0')
        assert version == '7.2.3'
```